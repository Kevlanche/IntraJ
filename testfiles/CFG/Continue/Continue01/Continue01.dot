digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[16]"; "List[17]"; "List[18]"; "Opt[19]"; "Exit[20]"}
{ rank = same; "List[15]"; "Block[20]"}
{ rank = same; "List[21]"}
{ rank = same; "ForStmt[22]"}
{ rank = same; "List[23]"; "Opt[24]"; "List[25]"; "Block[26]"; "ImplicitCondition[39]"}
{ rank = same; "List[27]"}
{ rank = same; "VarDeclStmt[28]"; "ContinueStmt[36]"}
{ rank = same; "Modifiers[29]"; "PrimitiveTypeAccess[31]"; "List[32]"; "List[38]"; "UncheckedExceptions[37]"}
{ rank = same; "List[30]"; "VariableDeclarator[33]"}
{ rank = same; "List[34]"; "Opt[35]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Continue01 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[16]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[17]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[18]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[19]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[20]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[21]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ForStmt[22]" [label="ForStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[23]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[24]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[25]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[26]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[27]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[28]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[29]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[30]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[31]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[32]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[33]" [label="VariableDeclarator\n < int c >", fillcolor=white   style=filled  ]
"List[34]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[35]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ContinueStmt[36]" [label="ContinueStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[38]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[37]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ImplicitCondition[39]" [label="ImplicitCondition\n < true >", fillcolor=white   style=filled  ]
"Exit[20]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[16]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[18]" [style=dashed, color=gray]
"ForStmt[22]" -> "List[23]" [style=dashed, color=gray]
"ForStmt[22]" -> "Opt[24]" [style=dashed, color=gray]
"ForStmt[22]" -> "List[25]" [style=dashed, color=gray]
"Modifiers[29]" -> "List[30]" [style=dashed, color=gray]
"VarDeclStmt[28]" -> "Modifiers[29]" [style=dashed, color=gray]
"VarDeclStmt[28]" -> "PrimitiveTypeAccess[31]" [style=dashed, color=gray]
"VariableDeclarator[33]" -> "List[34]" [style=dashed, color=gray]
"VariableDeclarator[33]" -> "Opt[35]" [style=dashed, color=gray]
"List[32]" -> "VariableDeclarator[33]" [style=dashed, color=gray]
"VarDeclStmt[28]" -> "List[32]" [style=dashed, color=gray]
"List[27]" -> "VarDeclStmt[28]" [style=dashed, color=gray]
"ContinueStmt[36]" -> "List[38]" [style=dashed, color=gray]
"ContinueStmt[36]" -> "UncheckedExceptions[37]" [style=dashed, color=gray]
"List[27]" -> "ContinueStmt[36]" [style=dashed, color=gray]
"Block[26]" -> "List[27]" [style=dashed, color=gray]
"ForStmt[22]" -> "Block[26]" [style=dashed, color=gray]
"ForStmt[22]" -> "ImplicitCondition[39]" [style=dashed, color=gray]
"List[21]" -> "ForStmt[22]" [style=dashed, color=gray]
"Block[20]" -> "List[21]" [style=dashed, color=gray]
"Opt[19]" -> "Block[20]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[19]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[20]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"VariableDeclarator[33]" -> "ImplicitCondition[39]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[33]" -> "ImplicitCondition[39]" [color=red, constraint=false]
"Exit[20]" -> "ImplicitCondition[39]" [color=red, constraint=false]
"ImplicitCondition[39]" -> "VariableDeclarator[33]" [color=blue, constraint=false, xlabel="T "] 
"ImplicitCondition[39]" -> "Exit[20]" [color=blue, constraint=false, xlabel="F "] 
"ImplicitCondition[39]" -> "Entry[13]" [color=red, constraint=false]
"ImplicitCondition[39]" -> "VariableDeclarator[33]" [color=red, constraint=false]
"Entry[13]" -> "ImplicitCondition[39]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[16]" -> "List[17]" -> "List[18]" -> "Opt[19]" -> "Exit[20]" [style=invis]
"List[23]" -> "Opt[24]" -> "List[25]" -> "Block[26]" [style=invis]
"VarDeclStmt[28]" -> "ContinueStmt[36]" [style=invis]
"Modifiers[29]" -> "PrimitiveTypeAccess[31]" -> "List[32]" [style=invis]
"List[34]" -> "Opt[35]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
41", fillcolor=1]
Edges [label="Total Nr. Edges
48", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
4", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
4", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}