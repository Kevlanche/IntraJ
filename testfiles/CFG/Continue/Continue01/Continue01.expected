NodeName: Entry[13] Successors: [ImplicitCondition[39]]
NodeName: ImplicitCondition[39] Successors: [VariableDeclarator[33], Exit[20]]
NodeName: VariableDeclarator[33] Successors: [ImplicitCondition[39]]
NodeName: Exit[20] Successors: []

NodeName: Exit[20] Predecessor: [ImplicitCondition[39]]
NodeName: ImplicitCondition[39] Predecessor: [Entry[13], VariableDeclarator[33]]
NodeName: Entry[13] Predecessor: []
NodeName: VariableDeclarator[33] Predecessor: [ImplicitCondition[39]]
